packages/elements/src/components/ino-tab-bar/ino-tab-bar.e2e.ts

```diff
@@ -19,6 +19,17 @@ describe('InoTabBar', () => {
       const activeTab = await tabBar.getAttribute('active-tab');
       expect(activeTab).toBe('3');
     });
+
+    it('should render with the autofocus property set correctly', async () => {
```

Comment:
This test only checks whether stencil sets the property correctly. I would suggest querying the focused element on that page and checking if it's the intended one. We also got a utility function you can use at `packages/elements/src/util/test-utils.ts`.

Here is an example: https://github.com/inovex/elements/blob/5167724116b44e8d384852b56a42177878dab59c/packages/elements/src/components/ino-input/ino-input.e2e.ts#L11-L27